src/cli/backup/backup.go

```diff
@@ -394,6 +394,10 @@ func genericDetailsCore(
 		return nil, clues.Wrap(errs.Failure(), "Failed to get backup details in the repository")
 	}
 
+	if len(d.Entries) == 0 {
+		return nil, clues.New("empty backup")
```

Comment:
nit, since "empty" can be interpreted many ways.
```suggestion
		return nil, clues.New("no items in backup")
```